### DISCUSSION TITLE
Hide 3D model while searching for surface in WebXR

### DISCUSSION THREAD

**Discussion by milesgreen** [google/model-viewer#2037] [Ideas]
Is there a way to hide the 3D model in WebXR while user is scanning room / searching for surface?

In testing, I've seen users stop at the 3D model preview, because it kind of animates with phone movement even though not placed in AR yet, and they think that's it. 

Ideally, I'd like to only reveal the model once it has been 'placed'. Maybe show the bounding box (either horizontal or vertical) while scanning?

**Comment by elalish** [comment]
I'm actually working on an instant-placement UX instead, as I've noticed the same thing. I'd rather do this than hide the model, as some environments make it very hard for ARCore to find a floor, and the user can get very annoyed if they don't see any model at all.

**Reply by milesgreen** [reply]
Awesome. Good point. 

Otherwise, direct access to manipulate model visibility and bounding box marker visibility in response to AR events would also work.
